### PR TITLE
[Feat]: Result generation after quiz attempt

### PR DESCRIPTION
**Pull request by parthpandyappp**
- Score evaluation
- Storing the latest scores in the db (To be used in next PR to list down the latest scores of the user)
- Displaying correct options as well as the one chosen by user.

**note**:
-  _Deploy preview below is usable as I've added that domain to the list of auth-domain in firebase_
- _All the logs for error handling will be replaced by toasts in future commits_

### CODE REVIEW

File: src/styles/login.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+.auth-form form .inp-unit {
+  display: flex;
+  flex-direction: column;
+  margin-top: 2rem;
+}
+
+.login-btn {
+  padding: 0.5rem;
+}
```
Comment (Sharmaryan) [inline src/styles/login.css:23]: you can use px here

**Review by Sharmaryan — APPROVED** [review]
- UI is simple and looking good
- change your app name, it is showing react app
- code is clean

File: src/backend/index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+import { getFirestore } from "firebase/firestore";
+
+const firebaseConfig = {
+    apiKey: process.env.REACT_APP_apiKey,
```
Comment (Logan1x) [inline src/backend/index.js:6]: Env Variable has been used, nice feat.

**Review by Logan1x — APPROVED** [review]
- Good use of Env to hide variables.
- use firebase for auth, nice.
- Meta tags are good for seo

**Pull request merged**